src/builder-manifest.c

```diff
@@ -2773,17 +2767,56 @@ builder_manifest_cleanup (BuilderManifest *self,
 
       if (self->appstream_compose && appdata_file != NULL)
         {
-          g_autofree char *basename_arg = g_strdup_printf ("--basename=%s", self->id);
-          g_print ("Running appstream-compose\n");
-          if (!appstream_compose (app_dir, error,
-                                  self->build_runtime ?  "--prefix=/usr" : "--prefix=/app",
-                                  "--origin=flatpak",
-                                  basename_arg,
-                                  self->id,
-                                  NULL))
+          g_autofree char *origin = g_strdup_printf ("--origin=%s",builder_manifest_get_id (self));
```

Comment:
style nit:
```suggestion
          g_autofree char *origin = g_strdup_printf ("--origin=%s", builder_manifest_get_id (self));
```

(same for elsewhere in this file)